Android/FirebaseJobDispatcher/source/Firebase.JobDispatcher/Firebase.JobDispatcher.csproj

```diff
@@ -0,0 +1,112 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Xamarin.Build.Download.0.4.2\build\Xamarin.Build.Download.props" Condition="Exists('..\packages\Xamarin.Build.Download.0.4.2\build\Xamarin.Build.Download.props')" />
```

Comment:
Hmmm, I do all my package management via the IDE. I'll update all the NuGet's and see if this cleans it up.